src/main/java/io/cdap/plugin/batch/KinesisSink.java

```diff
@@ -78,9 +76,7 @@ public void prepareRun(BatchSinkContext batchSinkContext) throws Exception {
 
     Schema schema = batchSinkContext.getInputSchema();
     if (schema != null && schema.getFields() != null) {
-      recordLineage(batchSinkContext, config.referenceName, schema,
-                    schema.getFields().stream().map(Schema.Field::getName).collect(Collectors.toList()),
-                    "Write", "Wrote to Kinesis Stream");
+      recordLineage(batchSinkContext, config.referenceName, schema, "Write", "Wrote to Kinesis Stream");
```

Comment:
Let's put this method in ReferenceBatchSink instead of cdap

Reply:
Ack. see https://github.com/cdapio/hydrator-plugins/pull/1071